.eslintrc.js

```diff
@@ -54,37 +54,50 @@ module.exports = {
 		'lines-between-class-members': [1],
 		'max-len': [0],
 		'newline-per-chained-call': [0, { 'ignoreChainWithDepth': 2 }],
-		'no-mixed-operators': [0],
+		'no-mixed-operators': [1],
 		'no-nested-ternary': [0],
-		'no-param-reassign': [0], // TODO: enable this check
+		'no-param-reassign': ['error', {
+			props: true,
+			ignorePropertyModificationsFor: [
+				'acc', // for reduce accumulators
+				'document', // for IIFE with window & document params
+				'c2pFrame', // for mutable iframe c2p element
+				'trackerEl', // for trackers.forEach()
+				'categoryEl', // for categories.forEach()
+				'dataEntry', // for data.forEach()
+				'appEntry', // for apps.forEach()
+				'sEntry', // for apps.forEach()
+				'confMutable', // for mutable conf settings
+				'eventMutable', // for mutable event details objects
+			]
+		}],
 		'no-plusplus': [0],
-		'no-prototype-builtins': [0], // TODO: enable this check
-		'no-restricted-syntax': [0], // TODO: enable this check
+		'no-prototype-builtins': [0],
+		'no-restricted-syntax': [1],
 		'no-tabs': [0],
 		'no-underscore-dangle': [0],
 		'no-unused-vars': [1],
 		'no-useless-escape': [1],
 		'operator-linebreak': [0],
-		'prefer-object-spread': [0], // TODO: enable this check
+		'prefer-object-spread': [1],
 		'space-before-function-paren': [2, 'never'],
-		'template-curly-spacing': [0],
 
 		// Plugin: Import
 		'import/no-cycle': [0],
-		'import/prefer-default-export': [0],
+		'import/prefer-default-export': [1],
 
 		// Plugin: React
-		'react/destructuring-assignment': [0],
+		'react/destructuring-assignment': [1],
 		'react/static-property-placement': [0],
 		'react/jsx-curly-newline': [0],
 		'react/jsx-indent': [1, 'tab'],
 		'react/jsx-indent-props': [1, 'tab'],
-		'react/jsx-props-no-spreading': [0], // TODO: enable this check
-		'react/no-access-state-in-setstate': [0], // TODO: enable this check
+		'react/jsx-props-no-spreading': [1],
+		'react/no-access-state-in-setstate': [1],
```

Comment:
Remove both jsx-props-no-spreading and no-access-state-in-setstate

Reply:
updated in feature/cleanup-cleanup as this PR was already merged.

Reply:
lgtm in feature/cleanup-cleanup